### PR TITLE
[FEAT] Water Deco

### PR DESCRIPTION
**Pull request by Rofrtd**
# Description

Add static wave image to land expansion background and a few water decorations.

Screen Shot 2022-07-07 at 7 12 56 pm

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [x] Screenshot if it includes any UI changes
- [x] I have read the contributing guidelines and agree to the T&Cs
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/features/game/expansion/components/Water.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      style={{
+        height: "inherit",
+        width: "calc(100%)",
+        left: "0",
+        top: "0",
+      }}
+      className="absolute"
+    >
+      {/* Above Land */}
+      <Shark side="top" />
```
Comment (adamhannigan) [inline src/features/game/expansion/components/Water.tsx:27]: The shark pixels appear much larger than the water pixels. For consistency we need to ensure that all of the pixels are the same size.

I would suggest scaling the shark down so it is the same size as some of the pixels you see on the land - such as the pebble or crop.

Screen Shot 2022-07-08 at 9 02 47 am

File: src/features/game/expansion/LandExpansion.tsx [hunk 2]
```diff
@@ -50,6 +51,10 @@ export const LandExpansion: React.FC = () => {
             // TODO dynamic game board size based on tile dimensions
           >
             <img src={background} className="absolute inset-0 w-full h-full" />
+            <div
+              className="absolute inset-0 bg-repeat w-full h-full"
+              style={{ backgroundImage: `url(${waterMovement})` }}
```
Comment (adamhannigan) [inline src/features/game/expansion/LandExpansion.tsx:56]: If you look closely, the water appears quite blurry. Consider adding the pixelated image rendering to the background image.

It also appears that the water pixels are smaller than the land pixels. We need to get the water pixel sizes perfect from the beginning as we will use this to size a range of objects on the water.

Try to scale the water so it correctly matches the pixel size of the land.

Screen Shot 2022-07-08 at 9 04 54 am

File: src/features/game/expansion/components/Water.tsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+      {/* Navigation Center Point */}
+      <div className="h-full w-full relative">
+        <span
+          id={Section.Water}
+          className="absolute left-1/2 top-1/2 transform -translate-x-1/2 -translate-y-1/2"
+        />
+
+        <img
+          src={dragonfly}
+          className="absolute left-2/3 top-1/2 animate-float"
```
Comment (adamhannigan) [inline src/features/game/expansion/components/Water.tsx:41]: Screen Shot 2022-07-08 at 9 02 06 am

Just another small one, but the dragonfly pixels are much larger than the water pixels

**Review by adamhannigan — CHANGES_REQUESTED** [review]
Nice work mate! The ocean is coming to life 🌊 

We just need to be consistent in the pixel sizes of the objects to make sure everything is perfect. We want to set a benchmark/standard for pixel perfect sizes now that we have a clean slate.

I've left a few suggestions around resizing some of the artwork to keep the pixel size consistent

**Comment by Z1kou** [comment]
Just an opinion, 

I would refrain from adding any swimming goblins / farmers to the farmers As I don't think it will fit the lore we want of getting haunted from a continent on an island and being extremly limited on human/gob ressource's 

Same thing with the dragon flies that would never exist near salty water, Maybe we replace the with other small sea creature's, like shrimps or shells ect

This is just me trying to put the lore of the game into action and since we are in a "magical" world all i said can be ommited  🤣

**Review by adamhannigan — APPROVED** [review]
Nice work mate! Looks much more consistent 💪

**Pull request merged**